jbpm-test-util/src/main/java/org/jbpm/test/persistence/util/PersistenceUtil.java

```diff
@@ -236,20 +234,19 @@ private static Properties getDefaultProperties() {
      * @return Properties containing the datasource properties.
      */
     public static Properties getDatasourceProperties() { 
-        String propertiesNotFoundMessage = "Unable to load datasource properties [" + DATASOURCE_PROPERTIES + "]";
         boolean propertiesNotFound = false;
 
         // Central place to set additional H2 properties
         System.setProperty("h2.lobInDatabase", "true");
         
         Properties props = new Properties();
         try (InputStream propsInputStream = PersistenceUtil.class.getResourceAsStream(DATASOURCE_PROPERTIES)){
-            assertNotNull(propertiesNotFoundMessage, propsInputStream);
             props.load(propsInputStream);
-        } catch (IOException ioe) {
+        } catch (Exception e) {
             propertiesNotFound = true;
```

Comment:
This property is only used below but combined with a condition that seems a little bit inconsistent (not sure why the password is compared); therefore, it could be removed, adding here the call to `getDefaultProperties`.
```suggestion
            props = getDefaultProperties();
```

Reply:
thanks for that! - done